Sources/ScreenObject/ScreenObject.swift

```diff
@@ -83,6 +93,21 @@ open class ScreenObject {
         return self
     }
 
+    @discardableResult
+    func waitForFirstScreenElement() throws -> Self {
+        try XCTContext.runActivity(named: "Confirm the first element in screen \(self) is loaded") { (activity) in
+                let result = waitFor(
+                    element: expectedElement,
+                    predicate: "isEnabled == true",
+                    timeout: self.waitTimeout
+                )
+            
+                guard result == .completed else { throw WaitForScreenError.timedOut }
```

Comment:
I was thinking of extracting this part and the similar one in `waitForScreen()` located right above into a separate method, but then I thought that this is technically "two lines", so I was not sure if it's worth it 🤔 .